lib/manager/actions/versions.js

```diff
@@ -589,6 +595,25 @@ export function setVersionIndex (
   }
 }
 
+export function setComparedVersionIndex (
```

Comment:
I think `setComparedVersion` is probably a better name. Although an index is an argument, the main point of this method is to set a compared version which doesn't necessarily need an index.

Reply:
For this, I mostly replicated the code that deals with the `feedVersion` prop, but I also agree it doesn't necessarily make sense to switch between index and actual object back and forth, so @landonreed let me know whether to switch the code to store the compared version object instead.

Reply:
Redundant prop removed in https://github.com/ibi-group/datatools-ui/pull/528/commits/03695eda1153b68d93751b1f0d27738edc2b41b1.